spec/support/fixtures/approvals/email_subscriptionsunsubscribe.approved.html

```diff
@@ -7,11 +7,11 @@
   <body>
     <div class="unsubscribed">
   <div class="unsubscribed__title">
-    You have been unsubscribed from DEV digest emails. 😔
+    You have been unsubscribed from DEV(local) digest emails. 😔
   </div>
   <br /><div class="unsubscribed__message">
     You will no longer receive emails like this. If you'd like to re-subscribe or manage your full email preferences, please
-    <a href="https://dev.to/settings/notifications">click here</a>.
+    <a href="http://localhost:3000/settings/notifications">click here</a>.
```

Comment:
This is pretty dependent on the EnvFile values, this the default values so this test will pass unless you have overridden the values. Not sure how to cater for this otherwise, as I cant reference the ApplicationConfig in this file..

Reply:
Seems like `URL.url('/settings/notifications')` may be the way to go here to get this one to work with people's defaults? Although when testing not sure why you'd override these

Reply:
I agree with @fdoxyz that `URL.url(/settings/notifications)` seems like the best approach here!

Reply:
Hmm I don't think this is possible because it's a plain html file.